src/Sarif.Multitool/Sarif.Multitool.csproj

```diff
@@ -40,12 +36,21 @@
       <AutoGen>True</AutoGen>
       <DependentUpon>MultitoolResources.resx</DependentUpon>
     </Compile>
+    <Compile Update="Rules\RuleResources.Designer.cs">
+      <DesignTime>True</DesignTime>
+      <AutoGen>True</AutoGen>
+      <DependentUpon>RuleResources.resx</DependentUpon>
+    </Compile>
```

Comment:
>Compile [](start = 6, length = 7)

When I created the new-style project files, I missed this. As a result, the embedded resource wasn't being updated when I edited the `.resx` file.